packages/@glimmer/compiler-delegates/src/module-unification/code-generator.ts

```diff
@@ -182,7 +182,7 @@ export default class MUCodeGenerator {
         relativePath
       );
 
-      return { module: relativePath, name };
+      return {...locator, module: relativePath, name };
```

Comment:
Seems like you can omit `name` here (and remove it from the above destructuring assignment altogether).